x/subspaces/handle_tx.go

```diff
@@ -0,0 +1,258 @@
+func (m *Module) handleMoveUserGroupEvent(tx *juno.Tx, event abci.Event) error {
+	subspaceID, err := GetSubspaceIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+	groupID, err := GetUserGroupIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	// Update the user group
+	return m.updateUserGroup(tx.Height, subspaceID, groupID)
+}
+
+// handleSetUserGroupPermissionsEvent handles the setting of permissions for an existing user group
+func (m *Module) handleSetUserGroupPermissionsEvent(tx *juno.Tx, event abci.Event) error {
+	subspaceID, err := GetSubspaceIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+	groupID, err := GetUserGroupIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	// Update the user group
+	return m.updateUserGroup(tx.Height, subspaceID, groupID)
+}
+
+// handleDeleteUserGroupEvent handles the deletion of an existing user group
+func (m *Module) handleDeleteUserGroupEvent(tx *juno.Tx, event abci.Event) error {
+	subspaceID, err := GetSubspaceIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+	groupID, err := GetUserGroupIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	// Update the user group
+	return m.db.DeleteUserGroup(tx.Height, subspaceID, groupID)
+}
+
+// --------------------------------------------------------------------------------------------------------------------
+
+// handleAddUserToGroupEvent handles the addition of a user to an existing user group
+func (m *Module) handleAddUserToGroupEvent(tx *juno.Tx, event abci.Event) error {
+	subspaceID, err := GetSubspaceIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+	groupID, err := GetUserGroupIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+	member, err := GetUserFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	return m.db.AddUserToGroup(types.NewUserGroupMember(subspaceID, groupID, member, tx.Height))
+}
+
+// handleRemoveUserFromGroupEvent handles the removal of a user from an existing user group
+func (m *Module) handleRemoveUserFromGroupEvent(tx *juno.Tx, event abci.Event) error {
+	subspaceID, err := GetSubspaceIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+	groupID, err := GetUserGroupIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+	member, err := GetUserFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	return m.db.RemoveUserFromGroup(types.NewUserGroupMember(subspaceID, groupID, member, tx.Height))
+}
+
+// --------------------------------------------------------------------------------------------------------------------
+
+// handleSetUserPermissionsEvent handles the setting of permissions for an existing user
+func (m *Module) handleSetUserPermissionsEvent(tx *juno.Tx, event abci.Event) error {
+	subspaceID, err := GetSubspaceIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+	user, err := GetUserFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	return m.updateUserPermissions(tx.Height, subspaceID, user)
+}
```

Comment:
I think it would be better to add section attribute to Desmos core in the future. What do you think?